scripts/utils.ts

```diff
@@ -6,49 +6,19 @@ import os from "os";
 import * as splToken from "@solana/spl-token";
 import { sha256 } from "@noble/hashes/sha256";
 import * as borsh from "borsh";
+export { OrderConfig, FeeConfig } from "../ts-common/common";
+import { OrderConfig, FeeConfig, AuctionData } from "../ts-common/common";
```

Comment:
```suggestion
import { OrderConfig, FeeConfig, AuctionData } from "../ts-common/common";
export { OrderConfig, FeeConfig };
```

Reply:
Or maybe it’s enough to just remove the line with export { OrderConfig, FeeConfig }, because I haven’t come across any code that requires exporting these types from this file instead of directly from common. Does that make sense?

Reply:
>I haven’t come across any code that requires exporting these types from this file instead of directly from common

[Here](https://github.com/1inch/solana-fusion-protocol/blob/main/scripts/fusion-swap/fill.ts#L16) is on such import.